### PR TITLE
Added speed limits checks

### PR DESCRIPTION
**Pull request by davidakachaos**
Added new checks for speed limits.
These checks are implemented in the PR on validator.

### CODE REVIEW

File: default.js [hunk 1]
```diff
@@ -715,4 +715,39 @@
 "202.severity": 'W', // TODO: change to error once tested
 "202.title": "BETA: No public connection for public segment (slow)",
 "202.problem": "The public segement is not connected to any other public segment",
-"202.solution": "Verify if the segment is meant to be a public accessible segment, or it should be changed to a private segment"
+"202.solution": "Verify if the segment is meant to be a public accessible segment, or it should be changed to a private segment",
+// speed limits
+"210.enabled": true,
+"210.severity": 'W',
```
Comment (berestovskyy) [inline default.js]: It's warning by default.

File: default.js [hunk 2]
```diff
@@ -715,4 +715,39 @@
...
+"211.solutionLink": "W:Global/Creating_and_editing_road_segments#Speed_limit",
+"212.enabled": true,
+"212.severity": 'W',
+"212.title": "Segment has no speed limit set from A to B",
+"212.problem": "Segment has no speed limit set from A to B",
+"212.solution": "Verify the speed limit on the segment and set it",
+"212.solutionLink": "W:Global/Creating_and_editing_road_segments#Speed_limit",
+"213.enabled": true,
+"213.severity": 'W',
+"213.title": "Segment has no speed limit from B to A",
```
Comment (berestovskyy) [inline default.js]: Missing word `set`

File: default.js [hunk 3]
```diff
@@ -715,4 +715,39 @@
...
+"213.problem": "Segment has no speed limit from B to A",
```
Comment (berestovskyy) [inline default.js]: Same here

**Comment by davidakachaos** [comment]
Thanks for your reviews, I'll work on these tomorrow :+1:

**Comment by berestovskyy** [comment]
Looks fine, thanks.

**Pull request merged**